src/test/java/com/solomka/controllers/BookControllerTest.java

```diff
@@ -23,20 +39,57 @@ public void createBookTest() throws Exception {
         createBookDto.setYearOfPublication(2020);
         createBookDto.setAuthors(Arrays.asList("author1", "author2"));
 
-        MockHttpServletRequestBuilder requestBuilder = MockMvcRequestBuilders.post("/books")
+        MockHttpServletRequestBuilder requestBuilder = MockMvcRequestBuilders.post(this.url)
                 .contentType(MediaType.APPLICATION_JSON_VALUE)
                 .content(OBJECT_MAPPER.writeValueAsString(createBookDto));
 
-        MvcResult result = mockMvc.perform(requestBuilder)
-                .andExpect(MockMvcResultMatchers.status().isCreated())
+        MvcResult result = this.mockMvc.perform(requestBuilder)
+                .andExpect(status().isCreated())
                 .andReturn();
 
         Book book = OBJECT_MAPPER.readValue(
                 result.getResponse().getContentAsString(),
                 Book.class
         );
 
-        mockMvc.perform(MockMvcRequestBuilders.get("/books/" + book.getBookId()))
-                .andExpect(MockMvcResultMatchers.status().isOk());
+        this.mockMvc.perform(get(url+"/" + book.getBookId()))
+                .andExpect(status().isOk());
+    }
+
+    @Test
+    public void getAllSuccessfulTest() throws Exception {
+        String expected = "{\"bookId\":\"random_id_value_5\",\"name\":\"7 habits of highly effective\",\"description\":\"\",\"authors\":[\"Stephen R Covey \"],\"yearOfPublication\":1989,\"numberOfWords\":464368,\"rating\":10}";
```

Comment:
would be better to use ObjectMapper dependency and use method 'write value as string')

Reply:
thank's. I will google it